wormhole-connect/src/components/TokensModal.tsx

```diff
@@ -184,45 +190,65 @@ function DisplayTokens(props: DisplayTokensProps) {
       <div className={classes.tokensContainer}>
         {tokens.length > 0 ? (
           <>
-            {tokens.map((token, i) => (
-              <div
-                className={classes.tokenRow}
-                key={i}
-                onClick={() => selectToken(token.key)}
-              >
-                <div className={classes.tokenRowLeft}>
-                  <TokenIcon name={token.icon} height={32} />
-                  <div>
-                    <div>{getDisplayName(token)}</div>
-                    <div className={classes.nativeChain}>
-                      {displayNativeChain(token)}
+            {tokens
+              // native tokens first
+              .sort((a, b) => {
```

Comment:
this is awesome. it might be nice to move to a `utils` function though, just to keep the component code smaller.

Reply:
changed